### PR TITLE
settings for deployment with heroku

### PR DESCRIPTION
**Pull request by wonderbeyond**
Here's my instance on heroku:
https://github-trending-api-wonder.herokuapp.com/repositories?language=python

### CODE REVIEW

**Comment by huchenme** [comment]
@wonderbeyond why do we need to include these configs since API is already deployed for everyone's usage? https://github-trending-api.now.sh/repositories

**Comment by wonderbeyond** [comment]
Just for one more option.

If you are using **now.sh**'s free plan, we may reach the limitation quickly. e.g. 1000 invocations per day.

Also **now.sh** maybe unavailable/unreachable for some user from some area. We need backups to make our apps more robust.

File: package.json [hunk 1]
```diff
@@ -18,6 +21,7 @@
     "clean": "rimraf dist",
     "rollup:build": "rollup -c",
     "rollup:build:prod": "NODE_ENV=production rollup -c",
+    "heroku-postbuild": "npm run rollup:build:prod",
```
Comment (huchenme) [inline package.json]: I don't think you need this line for API deployment.

**Comment by wonderbeyond** [comment]
For heroku-cli getting started, see https://devcenter.heroku.com/articles/heroku-cli

For normal nodejs deployment, see https://devcenter.heroku.com/articles/getting-started-with-nodejs
However this isn't a normal nodejs app, we need an extra build step using Rollup.

Heroku [has already respected the `yarn.lock` file to install dependencies before running our app](https://devcenter.heroku.com/articles/nodejs-support#build-behavior). And the `heroku-postbuild` script entry is to tell Heroku our [custom build process](https://devcenter.heroku.com/articles/nodejs-support#customizing-the-build-process).

File: package.json (hunk 1 above)
Comment (wonderbeyond) [reply package.json]: See my comment directly under this PR.

**Comment by wonderbeyond** [comment]
@huchenme If you want another heroku deployment:

- Visit [Heroku Dashboard](https://dashboard.heroku.com/apps/)
- Click [Create new app] link and select an available name
- Once successful, Heroku will maintain a git repo for you
- Add the remote url to your local git repo
- A git push will trigger a deployment

**Comment by huchenme** [comment]
@wonderbeyond merged, thanks for the contribution!

**Pull request merged**